python/az/aro/azext_aro/_params.py

```diff
@@ -49,10 +49,10 @@ def load_arguments(self, _):
                    validator=validate_client_secret(isCreate=True))
 
         c.argument('pod_cidr',
-                   help='CIDR of pod network.',
+                   help='CIDR of pod network. Should be a minimum of /18 or larger.',
```

Comment:
Is the correct world **Should** or would **Must** be better?  What happens if smaller than `/18`?

Reply:
Azure docs use _should_, but _must_ is probably a better word choice
https://docs.microsoft.com/en-us/azure/openshift/concepts-networking#networking--for-azure-red-hat-openshift

Reply:
I like MUST and AKS uses must for their POD cidr:
` This range must not overlap with any Subnet IP ranges. For example, 172.244.0.0/16.`

Reply:
Changes made and are reflective in upstream PR https://github.com/Azure/azure-cli/pull/18457